templates/lambda-template-nodejs/index.js

```diff
@@ -36,6 +36,7 @@ module.exports.handler = (event, context, cb) => {
       "configKeys": myVal
     };
 
+    logger.debug("Sample log for nodejs service");
```

Comment:
change to info. make changes in other files too

Reply:
Done